.env

```diff
@@ -0,0 +1,3 @@
+PUBLIC_URL=http://localhost:3000/
```

Comment:
This should go in `.env.sample`, please add `.env` to `.gitignore` and take it out of the commits.

Reply:
done!